src/modules/CRUD/components/Details/Read/AttachmentView/ImportFile/ImportFile.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Intent } from '@blueprintjs/core';
+import Uppy from '@uppy/core';
+import { Dashboard } from '@uppy/react';
+import { buildHeaders } from '@terralego/core/modules/Api';
+import locale from '@uppy/locales/src/fr_FR';
+import XHRUpload from '@uppy/xhr-upload';
+import classNames from 'classnames';
+
+import UppySetCategoryBeforeUpload from './UppySetCategoryBeforeUpload';
+import CategorySelector from '../CategorySelector';
+
+import '@uppy/core/dist/style.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import '@uppy/dashboard/dist/style.css';
+import './styles.scss';
+
+class ImportFile extends React.Component {
```

Comment:
Because it's the recommended way by the package https://uppy.io/docs/react/dashboard/#Initializing-Uppy

> A simple approach is to initialize it in your React component’s constructor() and destroy it in componentWillUnmount().

Anyway, the last week i read this PR, maybe we could convert it in functional component and use `useMemo` https://github.com/transloadit/uppy/pull/1936

Reply:
It's not the recommended way but a "simple approch". I think the doc was written before hooks and functional component ;-) 
I'll not insist, you can resolve this when you've read it.